src/test/airtable/test_airtable_round_winners.test.js

```diff
@@ -240,4 +249,31 @@ describe('Calculating Winners', function() {
         should.equal(partiallyFundedResults.length, 1)
         should.equal(notFundedResults.length, 3)
     });
+
+    it('Test new earmarks structure', function() {
+        //set earmarks for proposals and add USD Granted
+        allProposals[0].fields['Earmarks'] = Earmarks.NEW_OUTREACH
+        allProposals[0].fields['USD Requested'] = 2000
+
+        allProposals[1].fields['Earmarks'] = Earmarks.NEW_OUTREACH
+        allProposals[1].fields['USD Requested'] = 2000
+
+        allProposals[2].fields['Earmarks'] = Earmarks.NEW_OUTREACH
+        allProposals[2].fields['USD Requested'] = 2000
+
+        allProposals[3].fields['Earmarks'] = Earmarks.CORE_TECH
+        allProposals[3].fields['USD Requested'] = 2000
+
+        allProposals[4].fields['Earmarks'] = Earmarks.CORE_TECH
+        allProposals[4].fields['USD Requested'] = 2000
+
+        allProposals[5].fields['USD Granted'] = 2000
+
+        //filter all proposals that have Earmarks
+        let proposalsWithEarmark = allProposals.filter(p => p.get('Earmarks') !== undefined)
+
+        //calculate and get all winning proposals
+        let earmarkedResults = calculateWinningAllProposals(proposalsWithEarmark, fundingRound, oceanPrice)
+        should.equal(earmarkedResults.winningProposals.length, 5)
+    });
```

Comment:
Thank you for creating this new test Norbert. It really helps to review the commit, and to double check that the design is working as expected. I have reviewed the rest of the code lightly, and am basing my review mostly from this, which leads me to believe it's working as intended.